csharp/documentation/library-coverage/cwe-sink.csv

```diff
@@ -0,0 +1,2 @@
+CWE,Sink identifier,Label
+CWE-838,html,Inappropriate encoding
```

Comment:
CWE coverage might need some work: `html` sinks also count towards XSS.
I think we should remove the row for `CWE-838`, and add the XSS one instead, with `xss html` as sink identifiers. (This is not yet covered in the RST generation.)